demos/demo_runner/iot_demo_runner.c

```diff
@@ -70,6 +70,9 @@ int DEMO_entryFUNCTION( bool awsIotMqttMode,
     #define DEMO_networkDisconnectedCallback    ( NULL )
 #endif
 
+#ifdef democonfigMEMORY_ANALYSIS
+    demoMEMORY_ANALYSIS_STACK_DEPTH_TYPE xMemoryAnalysisStackSize;
```

Comment:
Happy to accept any suggestions for re-architecting how stack size is acquired. Ideally democonfigDEMO_STACKSIZE would just be used in iot_demo_freertos.c but this macro gets redefined in iot_demo_runner.h